keyboards/keeblounge/e60/readme.md

```diff
@@ -0,0 +1,14 @@
+# E60
+
+![E60](https://i.imgur.com/3IYT8Ne.jpg)
+
+A simple 5x12 pcb.
+
+* Keyboard Maintainer: [Digid3ar](https://github.com/DigiD3ar)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
could you add instructions on how to enter the bootloader?

Reply:
added on last commit